### PR TITLE
Fix IE8 issues

### LINKED ISSUE #26: IE8 issue
Following stuff fails in IE8:
- `IMG` tag
- data-attributes are not saved: there is no dataset

### PR DESCRIPTION
**Pull request by agentcooper**
Closes #26 

IMG tag and styles are working correct now.

For data-attributes I've made a manual solution, where you need to provide data attributes you will search for:

``` js
vdomVirtualize.searchForDataAttributes(['data-my-attr', 'data-option', 'data-id']);
// ...
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,6 +71,15 @@ function createFromElement(el) {
 function getElementProperties(el) {
   var obj = {}
 
+  // IE8 data attributes
+  if (dataAttrs.length > 0) {
+    dataAttrs.forEach(function(dataAttr) {
+      if (el.getAttribute(dataAttr)) {
+        obj[dataAttr] = el.getAttribute(dataAttr);
```
Comment (marcelklehr) [inline index.js]: Is this, how vdom wants data-attribs? Does this work? (I assumed it only accepted DOM properties, and  setting the name of an attrib as a property doesn't seem to make sense with that assumption in mind...)

Comment (agentcooper) [inline index.js]: It seems to work, here is the test case: https://github.com/agentcooper/vdom-virtualize/blob/master/specs/spec/spec.js#L55

**Comment by marcelklehr** [comment]
I found a simpler solution to enable data-* attributes, see #26

File: index.js [hunk 2]
```diff
@@ -75,15 +92,39 @@ function getElementProperties(el) {
...
 
       obj[propName] = css
       return
     }
 
+    if (el.tagName.toLowerCase() === 'textarea' && propName === 'type') {
+      return;
+    }
+
+    if (el.tagName.toLowerCase() === 'button' && propName === 'type') {
```
Comment (marcelklehr) [inline index.js]: Is this really necessary? I notice github has `<button type="button">`s...

**Pull request closed without merging**